### PR TITLE
Ground Nav markup

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

**Question:** What's below Sky Nav?

**Answer:** Ground Nav!

I'm using the same strategy for this pattern as the last (#687): Building the markup in a (hopefully) accessible way, getting that reviewed and merged, then tackling styles.

I've also included an example using the content from PWA Stats.

## Screenshots

Screen Shot 2020-05-29 at 2 28 52 PM

Screen Shot 2020-05-29 at 2 29 03 PM

## Testing

Take a look at the markup and structure and let me know if there's anything we could be doing to make it more inclusive.

---

- See #480

### CODE REVIEW

**Comment by derekshirk** [comment]
> **Question:** What's below Sky Nav?
> 
> **Answer:** Ground Nav!

🤣

**Comment by tylersticka** [comment]
> The `footer` should be identified as role `contentinfo` and should be the only content info landmark. However, in Safari, [VoiceOver incorrectly identifies the address within the footer as a second `contentinfo` landmark](https://www.scottohara.me/blog/2018/03/03/landmarks.html#address).

It looks like the consensus in the A11y Slack is to do nothing, but it did seem to inspire [a new WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=212617), so that's positive!

File: src/components/ground-nav/ground-nav.twig [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      <a href="tel:{{telephone|replace({
+        '.': '-',
+        '(': '-',
+        ')': '-',
+        ' ': ''
+      })}}">{{telephone}}</a>
+    </address>
+  {% endwith %}
+  {# https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html #}
+  <ul role="list">
```
[the remark this answers is not in the document]
Comment (tylersticka) [reply src/components/ground-nav/ground-nav.twig:61]: I consider it part of our contact information, personally.

**Comment by tylersticka** [comment]
@Paul-Hebert Only change I made to this was using the new icon component template (#768) instead of constructing the icon paths manually.

**Pull request merged**